go.mod

```diff
@@ -243,6 +244,8 @@ replace (
 	// override the go-scm from tekton
 	github.com/jenkins-x/go-scm => github.com/jenkins-x/go-scm v1.11.5
 
+	github.com/jenkins-x/jx-helpers => /Users/rochana/Code/Devops/repos/jx-helpers
```

Comment:
Ready for review? Then you want to remove this replace ...

Reply:
noted

Reply:
i updated the code and did some refactoring. please let me know if i can improve. This is my first pr for and OSS project.

Reply:
i updated the code and did some refactoring. please let me know if i can improve.